### PR TITLE
TangoRS fee changes

### PR DESCRIPTION
**Pull request by paullojorgge**

### CODE REVIEW

File: lib/Net/DRI/DRD.pm [hunk 1]
```diff
@@ -1454,7 +1454,7 @@ sub _build_price_query
  }
  my $fee_version = 0+($ndr->protocol()->{brown_fee_version} // $ndr->protocol()->{fee_version} // 0);
  # CentralNic::Fee (<=011) this extension is used in addition to premium_domain above, hense to elsif.
- if ((grep $_ eq 'Net::DRI::Protocol::EPP::Extensions::CentralNic::Fee', @{$ndr->protocol()->{loaded_modules}}) || $fee_version <= 0.11)
+ if ((grep $_ eq 'Net::DRI::Protocol::EPP::Extensions::CentralNic::Fee', @{$ndr->protocol()->{loaded_modules}}) || ( $fee_version <= 0.11 && $fee_version > 0 )  )
```
Comment (thedarkwinter) [inline lib/Net/DRI/DRD.pm:1457]: What does this additional condition do? Is fee_version somehow less that 0 sometimes?

File: lib/Net/DRI/DRD/NGTLD.pm [hunk 2]
```diff
@@ -1037,16 +1037,18 @@ L<Net::DRI::Protocol::EPP::Extensions::TangoRS::Auction> urn:ar:params:xml:ns:au
...
 L<Net::DRI::Protocol::EPP::Extensions::TangoRS::LaunchPhase> : http://xmlns.corenic.net/epp/mark-ext-1.0
 
-L<Net::DRI::Protocol::EPP::Extensions::CentralNic::Fee> urn:centralnic:params:xml:ns:fee-0.6
-
-Fee extension is currently only used in .NRW and for domain_check command only.
-
 =cut
 
+ my $tangorsversion; # setting this variable because this profile use distinct fee extensions per TLD :(
+ $tangorsversion = undef if $tld eq 'ruhr'; # no fee extension
```
Comment (thedarkwinter) [inline lib/Net/DRI/DRD/NGTLD.pm:1043]: Should this not be default? I.e. set to undef for all, not just for 'ruhr', and then only set for specific TLDs? In this case, gmx wouldn't be set to anything ...

File: lib/Net/DRI/Protocol/EPP/Extensions/Fee.pm [hunk 3]
```diff
@@ -155,7 +155,11 @@ sub setup
   # This means any commands called before greeting will use that version until its bumped to highest version
   my $v = $po->{brown_fee_version} // $po->{fee_version};
   $v = '1.0' unless defined $v && $v =~ m/^\d.(\d+)$/;
-  $po->ns({ 'fee' => [ 'urn:ietf:params:xml:ns:fee-'.$v,'fee-'.$v.'.xsd' ] });
+  if ($v && $v eq '1.0') {
+    $po->ns({ 'fee' => [ 'urn:ietf:params:xml:ns:epp:fee-'.$v,'fee-'.$v.'.xsd' ] });
+  } else {
+    $po->ns({ 'fee' => [ 'urn:ietf:params:xml:ns:fee-'.$v,'fee-'.$v.'.xsd' ] });
```
Comment (thedarkwinter) [inline lib/Net/DRI/Protocol/EPP/Extensions/Fee.pm:161]: lol not sure how i missed that :)

**Review by thedarkwinter — CHANGES_REQUESTED** [review]
Thanks @paullojorgge . I have two minor comments. If you can please clarify those, but should be not problem anyway :+1:

File: lib/Net/DRI/Protocol/EPP/Extensions/Fee.pm (hunk 3 above)
Comment (paullojorgge) [reply lib/Net/DRI/Protocol/EPP/Extensions/Fee.pm:161]: :)

File: lib/Net/DRI/DRD.pm (hunk 1 above)
Comment (paullojorgge) [reply lib/Net/DRI/DRD.pm]: added by me at the time when testing incorrect domain check price command :p

**Review by thedarkwinter — APPROVED** [review]

**Pull request merged**